iron-overlay-behavior.html

```diff
@@ -326,6 +312,12 @@
         return;
       }
 
+      if (!this.restoreFocusOnClose) {
+        this.restoreFocusNode = null;
+      } else if (this.opened) {
+        this.restoreFocusNode = this._manager.deepActiveElement;
+      }
+
       this._manager.trackBackdrop(this);
```

Comment:
Explore calling backdropElement.open/close only in here.
Open will call prepare internally?